viewing_party/party.py

```diff
@@ -0,0 +1,145 @@
+def create_movie(title,genre,rating):
+    mydict = {"title":title, "genre":genre,"rating":rating}
+    for value in mydict.values():
+        if value == None:
+            return None            
+    return mydict
+
+def add_to_watched(user_data, movie):
+    for key in user_data:       
+        user_data[key].append(movie)
+    return user_data 
+
+def add_to_watchlist(user_data, movie):
+    for key in user_data:   
+        user_data[key].append(movie)
+    return user_data 
+    
+
+def watch_movie(user_data, title):
+    for movie in user_data['watchlist']:   
+        if movie['title'] == title:   
+            user_data['watched'].append(movie)   
+            user_data['watchlist'].remove(movie)
+    return user_data
+
+
+def get_watched_avg_rating(janes_data):
+    count = 0
+    total = 0
+    if janes_data["watched"] == []:
+        return 0
+    for item in janes_data["watched"]:
+        total += item["rating"]
+        count += 1
+    average_rating = total/count
+    return  average_rating 
+ 
+
+def get_most_watched_genre(janes_data):
+    frequencey_genre = {}
+    max_value = 0
+    
+    for item in janes_data["watched"]:
+        temp = item["genre"]
+        if temp in frequencey_genre:
+            frequencey_genre[temp] +=1
+        else:
+            frequencey_genre[temp]= 1
+
+    for value in frequencey_genre.values():
+        if max_value < value:
+            max_value = value
+        
+    for key in frequencey_genre:
+        if frequencey_genre[key] == max_value:
+            return key
+
+
+def get_unique_watched(amandas_data):
+    unique_watched = []
+    amandas_watched = amandas_data["watched"]
+    amandas_friends_watched =amandas_data["friends"][0]["watched"]
+    for item in amandas_data["friends"][1]["watched"]:
+        if item not in amandas_friends_watched:
+            amandas_friends_watched.append(item)
+    
+    for item in amandas_watched:
+        if item not in amandas_friends_watched:
+            unique_watched.append(item)
+
+    return unique_watched
+    
+def get_friends_unique_watched(amandas_data):
+    unique_watched = []
+    amandas_watched = amandas_data["watched"]
+    amandas_friends_watched =amandas_data["friends"][0]["watched"]
+    for item in amandas_data["friends"][1]["watched"]:
+        if item not in amandas_friends_watched:
+            amandas_friends_watched.append(item)
```

Comment:
This section (75-79) is the same as 61-65 above.  I recommend pulling this out into a helper function.